waffle_configs/bedrock-prod.env

```diff
@@ -13,44 +13,26 @@ FUNNELCAKE_139_LOCALES=en-US
 FUNNELCAKE_139_PLATFORMS=win
 
 # switches
-SWITCH_BETA_WHATSNEW_68_TRAILHEAD=on
 SWITCH_CONTENTFUL_HOMEPAGE_DE=off
 SWITCH_CONTENTFUL_HOMEPAGE_EN=on
-SWITCH_CONTRIBUTE_SURVEY=off
 SWITCH_CONVERT_EXPERIMENT_SCRIPT=on
-SWITCH_DATA_PRIVACY_DAY_2017=on
-SWITCH_DEV_WHATSNEW_68=on
-SWITCH_DEV_WHATSNEW_68_TRAILHEAD=on
 SWITCH_DISABLE_SHA1_DOWNLOADS=off
-SWITCH_ELECTION_BUNDLE=on
-SWITCH_EXPERIMENT_FIREFOX_NEW_TRAILHEAD=off
-SWITCH_EXPERIMENT_FIREFOX_WHATSNEW_70=off
 SWITCH_EXPERIMENT_FIREFOX_WHATSNEW_96_EN=on
 SWITCH_FIREFOX_APP_STORE_BANNER=on
-SWITCH_FIREFOX_ELECTION_BUILD=on
-SWITCH_FIREFOX_DAYLIGHT_LAUNCH_BANNER=off
-SWITCH_FIREFOX_DAYLIGHT_PROMO_BANNER=on
-SWITCH_FIREFOX_DEV_FALL=on
-SWITCH_FIREFOX_LITE_WHATSNEW=on
 SWITCH_FIREFOX_MOBILE_BANNER=on
-SWITCH_FIREFOX_MR1_LAUNCH=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION_MODAL=on
 SWITCH_FIREFOX_YANDEX=on
-SWITCH_FIREFOX_PRE_DOWNLOAD_NEWSLETTER=off
 SWITCH_FUNDRAISING_BANNER_EOY2021=off
-SWITCH_HOMEPAGE_MR2_PROMO_EU=on
 SWITCH_LIFECYCLE_PAGE1_SURVEY=on
-SWITCH_NEW_REDESIGN=on
```

Comment:
Removed in https://github.com/mozilla/bedrock/commit/1970f400b0db15779fc83867c43025bb0727631d